spark-jobs/src/main/resources/application.conf.template

```diff
@@ -17,6 +17,10 @@ enceladus.version=${project.version}
 # each can have multiple comma-separated hosts, these are used for fault-tolerance
 menas.rest.uri="http://localhost:8080,host2:9000/menas;https://localhost:8080,host2:9000/menas"
 
+# 'enceladus_record_id' with an id can be added containing either true UUID, always the same UUIDs (pseudo) or the column
+# will not be added at all. Allowed values: "no", "pseudo", "true"
+enceladus.recordid.generation.strategy="true"
```

Comment:
@Zejnilovic, I hope you are ok with this form of overridability. 😉

Reply:
this is perfect 👍